content/release-10-10-0/oee-10-10-0-bundle/10_10_0_3.md

```diff
@@ -0,0 +1,187 @@
+---
+weight: 35
+title: Release 10.10.0.3
+layout: redirect
+---
+This release of the OEE App is addressing UI fixes and improvements as well as fixes of the OEE calculation.
+
+### Improvements
+
+<table>
+<colgroup>
+    <col style="width: 15%;">
+    <col style="width: 70%;">
+    <col style="width: 15%;">
+</colgroup>
+<thead>
+<tr>
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">Until now, all profiles needed a 10 min (600s) default calculation interval. If this interval was not configured for the calculation profile, it was automatically added by OEE application. With this change, it is now possible to define a default calculation interval for each profile, which will be used on the machine park overview. Calculation profiles created before this change, will be migrated to use the 10 min interval as their default interval.<br/><br/>With this change it is now also possible to reduce the number of MEAs created for the calculcation profiles. For slow producing machines, it might for example not be required to create a measurement every 10 mins.</td>
```

Comment:
```suggestion
<td style="text-align:left">Until now, all profiles needed a 10 min (600s) default calculation interval. If this interval was not configured for the calculation profile, it was automatically added by OEE application. With this change, it is now possible to define a default calculation interval for each profile, which will be used on the machine park overview. Calculation profiles created before this change, will be migrated to use the 10 min interval as their default interval.<br/><br/>With this change it is now also possible to reduce the number of MEAs created for the calculcation profiles. For slow producing machines, it might, for example, not be required to create a measurement every 10 mins.</td>
```